### PR TITLE
Create user-systemd-unit-on-boot.adoc

### PR DESCRIPTION
**Pull request by JaimeMagiera**
Addressing #205

### CODE REVIEW

File: modules/ROOT/pages/tutorial-user-systemd-unit-on-boot.adoc [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    - path: /home/sleeper/.config/systemd/user/linger-example.service 
+      mode: 0744
+      contents:
+        inline: |
+          [Unit]
+          Description=A systemd user unit demo
+          After=network-online.target
+          Wants=network-online.target
+          [Service]
+          ExecStart=/usr/bin/bash sleep infinity
```
Comment (lucab) [inline modules/ROOT/pages/tutorial-user-systemd-unit-on-boot.adoc:32]: I'm surprised by this line, does it actually work? I was naively expecting a simpler `/usr/bin/sleep infinity`.

**Comment by lucab** [comment]
Thanks for the PR, this short tutorial looks great! I've noted a minor doubt in review. Feel free to squash all the commits together.

File: modules/ROOT/pages/tutorial-user-systemd-unit-on-boot.adoc (hunk 1 above)
Comment (JaimeMagiera) [reply modules/ROOT/pages/tutorial-user-systemd-unit-on-boot.adoc]: Heh, no it does not. That was a tired eyes cut-and-paste error. Fixed and squashed.

**Review by lucab — APPROVED** [review]

**Comment by travier** [comment]
This is great. Thanks!

**Pull request merged**